app/resources/views/pages/home.blade.php

```diff
@@ -32,29 +32,29 @@
 		<div class="col-lg-7">
 			<div class="home-inputs">
 				@if (!$is_authenticated)
-				<div class="signin-signup">
-					<a href="/signin">
-						<table>
-							<tr>
-								<td class="user-icon">
-									<i class="fa fa-user"></i>
-								</td>
-								<td class="text-center sign-in-sign-up">
-									Sign in 
-								</td>
-							</tr>
-						</table>
-					</a>
+					<div class="signin-signup">
```

Comment:
What are these home.blade.php changes trying to fix or improve?  If it is just changes to indentation, I'd rather they get undone so that commands like 'git blame home.blade.php' will show more meaningful commits and reasons for those lines to change.